### DISCUSSION TITLE
Gps time sync

### DISCUSSION THREAD

**Discussion by roo12312** [gopro/labs#1112] [Q&A]
Hi,
Does the gps time sync also update the time of the go pro? 
Or its just used as timecode?

**Comment by dnewman-gpsw** [comment]
Updates both

**Reply by roo12312** [reply]
I have it enabled on my GoPro 10 but I see that the time is not being updated

**Reply by dnewman-gpsw** [reply]
It requires GPS lock.

**Reply by roo12312** [reply]
I didn't understand... Am I supposed to update any settings?

**Reply by dnewman-gpsw** [reply]
Scan *SYNC=1 once.  This camera clock will be set every time the GPS Lock occurs, when outside is typically with 60s of power on.

**Comment by dnewman-gpsw** [comment]
Here is a demo video of it working.  I used a HERO10, but 11 or 13 do this too.  I set the clock slightly wrong.  Scanned *ZONE=-420*SYNC=1  <- Zone helps, although not needed, this is the minutes offset from UTC time.  

https://github.com/user-attachments/assets/5902f16e-9aa6-4065-be01-3f2f17d2feef

So that you can see the clock update, here is a script that shows the clock and GPS DOP precision at the same time. 
![Clock+GPSDOP](https://github.com/user-attachments/assets/63fed50f-de85-4c4d-abcb-9d8d34dfd6d5)

https://gopro.github.io/labs/control/set/?cmd==St:S=Mt:N=Ht:H=TS=NM=IH=T/10=N/10=I/10=H%2510=M%2510=S%2510=I%257=N%257=T%257%22$I$H:$N$M:$T$S\nGPS%20DOP%20$d%22!R&title=GPS%20DOP%20and%20Time

**Comment by rkanaga** [comment]
How accurate is GPS sync vs doing a QR sync from the Tentacle Timber App at the beginning of the session/ day?

I find that Timebar sync is pretty good although not as accurate as wired sync from a Tentacle device, however I would prefer not to run any additional cables to the GoPro if possible. Is it correct that the GPS sync re calibrates for each recording and so might be better than a single Timber QR sync at the beginning of the session?

Thank you

**Reply by dnewman-gpsw** [reply]
GPS is a typically ±2 frames accurate, and it doesn't drift as it always updating.
